model-optimizer/extensions/ops/layer_norm.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from mo.front.common.partial_infer.elemental import copy_shape_infer
+from mo.graph.graph import Graph
+from mo.ops.op import Op
+
+
+class LayerNorm(Op):
```

Comment:
Could you write the more detail description of the operation?

Reply:
Could you write more detail description of the operation, please? More precisely, attributes of operations, conditions on input shapes, dimensions that are used to calculate, etc.